### DISCUSSION TITLE
에트리 하계연구연수생 서류합격 후기

### DISCUSSION THREAD

**Discussion by chairoi** [BDD-CLUB/challenge-every-week-posting#74] [4. 기타 글 모음]
서류 합격하게 되어서,,, 간단하게 후기 써 봤습니다!!
최종합격 공지가 빨리 나왔으면 좋겠네요.

[글 보러 가기](https://thisischacha.tistory.com/5)

**Comment by amaran-th** [comment]
헐 대박!!! 완전 멋져요 👍👍 축하드립니당!!!
최종합격되면 어디로 가는건가요??

**Comment by shkisme** [comment]
축하함니다~! 🎉🎉🎉

**Comment by llddang** [comment]
우왕! 축축!!!
최종합격 되면 좋겠다!!!
축하해!

**Comment by pipisebastian** [comment]
우왕!!! 최종합격 축하합니당~~!~!

**Comment by publdaze** [comment]
축하해요 🎉🎉🎉
